api/src/test/java/com/epam/pipeline/test/acl/AclTestBeans.java

```diff
@@ -17,47 +17,79 @@
 package com.epam.pipeline.test.acl;
```

Comment:
Could you please check if all of the changes in this class are relevant to the pull request? It seems that mocks like `PipelineDao` have nothing to do with acl tests.